iotdevice/transport/http/http.go

```diff
@@ -14,9 +14,12 @@ import (
 
 	"github.com/amenzhinsky/iothub/common"
 	"github.com/amenzhinsky/iothub/iotdevice/transport"
+	"github.com/amenzhinsky/iothub/iotservice"
 	"github.com/amenzhinsky/iothub/logger"
 )
 
+const API_VERSION = "2020-09-30"
```

Comment:
Please rename it to `apiVersion` there's no need to export it.

Reply:
fixed!